include/deal.II/distributed/grid_refinement.h

```diff
@@ -157,14 +181,34 @@ namespace parallel
        * processors, no cells are refined at all.
        *
        * The same is true for the fraction of cells that is coarsened.
+       *
+       * @param[in,out] tria The triangulation whose cells this function is
+       * supposed to mark for coarsening and refinement.
+       *
+       * @param[in] criteria The refinement criterion computed on each mesh cell
+       * active on the current triangulation. Entries may not be negative.
+       *
+       * @param[in] top_fraction_of_error The fraction of the total estimate
+       * which should be refined. If this number is zero, no cells will be
+       * refined. If it equals one, the result will be flagging for global
+       * refinement.
+       *
+       * @param[in] bottom_fraction_of_error The fraction of the estimate
+       * coarsened. If this number is zero, no cells will be coarsened.
+       *
+       * @param[in] norm_type Sets the type of norm to which the criteria of
+       * several cells will be concatenated. Currently
+       * VectorTools::NormType::L1_norm and VectorTools::NormType::L2_norm are
+       * supported.
```

Comment:
Here and in the other place, I don't think what "concatenated" is supposed to refer to. I think it would be useful if you stated what the refinement criterion actually is, and where these exponents come into play.

Reply:
I felt this was appropriate:
To determine thresholds, combined errors on subsets of cells are calculated as norms of the criteria on these cells.